OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -63,21 +81,12 @@ public void StartGame(Arguments args)
 
 			// Join a server directly
 			var connect = string.Empty;
-			if (args != null)
-			{
-				if (args.Contains("Launch.Connect"))
-					connect = args.GetValue("Launch.Connect", null);
 
-				if (args.Contains("Launch.URI"))
-				{
-					connect = args.GetValue("Launch.URI", null);
-					if (connect != null)
-					{
-						connect = connect.Replace("openra://", "");
-						connect = connect.TrimEnd('/');
-					}
-				}
-			}
+			if (!string.IsNullOrEmpty(Launch.Connect))
```

Comment:
If we're going to have the LaunchArgs class, then this logic really should be encapsulated inside it.